client/styles/Home.module.css

```diff
@@ -0,0 +1,121 @@
+.container {
```

Comment:
wat dis?

Reply:
skeleton?

Reply:
Boilerplate code. Keep for now for @mkshin1 to reference.

Reply:
ty